recipes/adios2/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Do you need this file? I think it should be in the source tarball

Reply:
I thought we should always add the LICENSE file separately?

Reply:
We always need to package it but we prefer to get it from source tarball(s) instead of including it in the recipe directory if we can to ensure it's correct and up to date.